src/commands/login/list-users-in-group.ts

```diff
@@ -13,21 +14,43 @@ export class ListUsersInGroup extends BaseCommand<typeof ListUsersInGroup> {
     'group-name': Flags.string({
       summary: 'Name of the user group',
     }),
+    'max-items': Flags.integer({
+      summary: "The total number of items to return in the command's output",
+      max: MAX_ITEMS_LIMIT,
```

Comment:
Should the max limit be defined in the CLI? Seems more of a restriction that should be in place in the API itself.
Or are you configuring the default? In which case we should just use the APIs default (not set it) unless that doesn't work for some reason.

Reply:
I removed the limitation of the flag, but I set the default to 15.

Reply:
The limit set in the API is 100